automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -167,63 +169,65 @@ trait GPAllocBeforeAndAfterAll extends GPAllocUtils with BeforeAndAfterAll {
         }
       } else loggerIO.info(s"Not going to release project: ${workspaceNamespaceProp} due to error happened")
       _ <- IO(sys.props.subtractAll(List(googleProjectKey, workspaceNamespaceKey, workspaceNameKey)))
-      _ <- ProxyRedirectClient.stopServer()
+      _ <- ProxyRedirectClient.stopServer(sys.props.get(proxyRedirectServerPortKey).get.toInt)
       _ <- loggerIO.info(s"Stopped proxy redirect server")
       _ <- IO(super.afterAll())
     } yield ()
 
-    res.unsafeRunSync()
+    res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
   // NOTE: createInitialRuntime / deleteInitialRuntime exists so we can ensure that project-level
   // resources like networks, subnets, etc are set up prior to the concurrent test execution.
   // We can remove this once https://broadworkbench.atlassian.net/browse/IA-2121 is done.
 
   private def createInitialRuntime(project: GoogleProject): IO[Unit] =
-    LeonardoApiClient.client.use { implicit c =>
-      implicit val authHeader = Authorization(Token(AuthScheme.Bearer, ronCreds.makeAuthToken().value))
-      for {
-        res <- LeonardoApiClient
-          .createRuntimeWithWait(
-            project,
-            initalRuntimeName,
-            LeonardoApiClient.defaultCreateRuntime2Request
-          )
-          .attempt
-        _ <- res match {
-          case Right(_) =>
-            loggerIO.info(s"Created initial runtime ${project.value} / ${initalRuntimeName.asString}")
-          case Left(err) =>
-            loggerIO
-              .warn(err)(
-                s"Failed to create initial runtime ${project.value} / ${initalRuntimeName.asString} with error"
-              )
-        }
-      } yield ()
-    }
+    if (isHeadless) {
```

Comment:
This means we only create the initial runtime for Jenkins runs, not local, right? That makes sense I think

Reply:
yep yep...I'm commenting it out every time I run it locally, so figured I'll make it a bit simpler :)..sry missed this in description